src/EnergyPlus/AirflowNetworkBalanceManager.hh

```diff
@@ -232,6 +233,63 @@ namespace AirflowNetworkBalanceManager {
 	Real64
 	GetZoneInfilAirChangeRate( int const ZoneNum ); // hybrid ventilation system controlled zone number
 
+	// derived class or struct
+	struct OccupantVentilationControlProp {
+
```

Comment:
If possible, could we make a conference call. Please let me know the available time.

 

 

From: Jason DeGraw [mailto:notifications@github.com] 
Sent: Thursday, February 19, 2015 5:09 PM
To: NREL/EnergyPlus
Cc: Lixing Gu
Subject: Re: [EnergyPlus] Advanced afn (#4729)

 

In src/EnergyPlus/AirflowNetworkBalanceManager.hh <https://github.com/NREL/EnergyPlus/pull/4729#discussion_r25031127> :

> @@ -232,6 +233,63 @@ namespace AirflowNetworkBalanceManager {
>       Real64
>       GetZoneInfilAirChangeRate( int const ZoneNum ); // hybrid ventilation system controlled zone number
>  
> +     // derived class or struct
> +     struct OccupantVentilationControlProp {
> +

@lgu1234 <https://github.com/lgu1234>  Declaring OpeningStatus as an OpenStatus enum rather than as an int is what you need to do:

OpenStatus OpeningStatus;

So the name is really not great, kind of confusing. Can you think of a more descriptive name for the enum? I tried and failed.

We definitely don't want to be doing the int() conversion.

—
Reply to this email directly or view it on GitHub <https://github.com/NREL/EnergyPlus/pull/4729/files#r25031127> .